node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vectordb",
-  "version": "0.1.0",
+  "version": "0.1.0-linux",
```

Comment:
you are checking the system in `node/native.js`, do you still need a system name in version tag